msctl

```diff
@@ -1407,7 +1407,7 @@ overviewer() {
             "color" : "aqua",
             "clickEvent" : {
               "action" : "open_url",
-              "value" : "'$MAPS_URL'/'$1'"
+              "value" : "'$MAPS_URL'"
```

Comment:
My bad again, I had changed it locally for my own server to point to somewhere else